README.md

```diff
@@ -73,49 +117,56 @@ Chrome is set per default. Firefox and Opera are supported.
 
 ### Creating JavaScript output
 
-To use the web-app, a "transpiled" version of the JavaScript code must
+A "transpiled" version of the JavaScript code must
 be available in the `UI/js/` directory. This is created from `UI/js-src/`
-by running
+by running either of the two commands shown below.
+
+```bash
+make jsdev # With VUE debugger enabled
+```
+or
 
 ```bash
-make js
+make js  # Without VUE debugger enabled
 ```
 
-After editing the code in `UI/js-src/`, this command needs to be re-run.
+After editing the code in `UI/js-src/`, one of these commands needs to be re-run.
 
 ### Accessing LedgerSMB
 
 As per the example above, you should be able to browse to your host at
 <http://host:32452/setup.pl> if you want to go through the proxy or at
-<http://172.20.0.6/setup.pl> to to go directly and create a test database.
+<http://172.20.0.6/setup.pl> to go directly and create a test database.
 The default password of the `postgres` user is `abc`.
 
-Similarly, when a test company exists, browsing to
+Similarly, after you create a test company using `setup.pl`, browsing to
 <http://172.20.0.6/login.pl> allows to log into the company.
 
-The postgres database is made available at <http://host:45632> on the example
+The Postgres database is made available at <http://host:45632> on the example
 above, should you want to browse it.
 
 All host ports are assigned randomly on available ports to not clash with the
-host but this can be overriden
+host but this can be overridden
 
 LedgerSMB offers the possibility to run in development mode, where you can
 see the modifications done in the user interface code be installed and shown
 in realtime.
 
 This is started with the command:
 
-* `npm run serve`\
-   Runs the realtime user interface compiler.
+```sh
+# Runs the realtime user interface compiler
+docker exec -t -d ldmaster_lsmb npm run serve
```

Comment:
make serve, to stay coherent

Reply:
Works for me.

Reply:
On further review, `make serve` does not work in scripts because `make` uses `docker exec -ti` instead of `docker exec -td`. So I'll just add a note that manual invocations can use `make serve`.